src/style.css

```
@@ -0,0 +1,83 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  min-width: 1024px;
+  max-width: 1200px;
+  
+}
+
+header {
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+  justify-content: flex-end;
+  position: relative;
+
+  margin-right: 50px;
+  
+  
+}
+
+.link-box {
```

Вместо того чтобы писать отдельный класс для `:last-child`
```suggestion
.link-box:not(:last-child) {
```